src/extlibs/er_force_sim/src/amun/simulator/simball.h

```diff
@@ -91,6 +99,17 @@ class camun::simulator::SimBall : public QObject
     btRigidBody *m_body;
     btMotionState *m_motionState;
     sslsim::TeleportBall m_move;
+    double rolling_speed    = -1;
+    bool setTransitionSpeed = true;
+
+    enum BallState
+    {
+        STATIONARY,
+        ROBOT_COLLISION,
+        SLIDING,
+        ROLLING
+    };
+    BallState currentBallState = STATIONARY;
```

Comment:
these variables should be initialized in the constructor

Reply:
done